gems/smithy/lib/smithy/views/client/operation_examples.rb

```diff
@@ -99,7 +99,7 @@ def structure(json, shape, indent)
             entry = value(value, member_shape, "#{indent}  ")
             lines << "#{indent}  #{key.underscore}: #{format(entry)},"
           end
-          lines.last.chomp!(',')
+          lines.last.chomp!(',') if lines.last.end_with?(',')
```

Comment:
Do you need `&.chomp`? can it ever be empty?  Also, why do you need to check if it ends_with?  chomp!, `'test'.chomp(',') just returns the same string`

Reply:
If there are no members, lines is just '{' and it's a frozen string literal, so there is a failure.